src/wovnio/html/HtmlConverter.php

```diff
@@ -415,7 +415,7 @@ public function _removeForm($node, $marker)
 
         if (strtolower($node->tag) === 'input' && strtolower($node->getAttribute('type')) == 'hidden') {
             $originalText = $node->getAttribute('value');
-            if (!$originalText || strpos($originalText, HtmlReplaceMarker::$keyPrefix) !== false) {
+            if ((!$originalText && $originalText !== '0') || strpos($originalText, HtmlReplaceMarker::$keyPrefix) !== false) {
```

Comment:
How about check with string length like `strlen($originalText) > 0` ?

Reply:
Agree, that's better. Fixed in e9831e79e8b11923bae7cfece62da0292451c9ae.

Reply:
I don't really like this either. What does `getAttribute` return when the attribute doesn't exist?

Reply:
Checking explicitly for `false` was what I originally had in 76efbbd3cc6b2166a587340bf34b55ae995b20a6. But I changed it to be more permissive to other falsy values. In the end, I don't think that's correct since what we want is to check when the attribute doesn't exist, as you said.

Fixed in 566846b4b91384ec31e062b455d510a98e3a2eba.